### PR TITLE
[14.0] [mig] pos_supplierinfo_barcode

### PR DESCRIPTION
**Pull request by fshah-initos**

### CODE REVIEW

**Comment by dsolanki-initos** [comment]
it would be nice if someone add review here.

**Review by dsolanki-initos — APPROVED** [review]
Code review and functional test LGTM

File: pos_supplierinfo_barcode/models/product.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+    # technical field used in POS frontend
+    supplier_barcode_json = fields.Char(
+        "Supplier barcode list", readonly=True, compute="_compute_supplier_barcode_json"
+    )
+
+    def _compute_supplier_barcode_json(self):
+        for t in self:
+            supplier_barcode_json = [x for x in t.mapped("seller_ids.barcode") if x]
+            t.supplier_barcode_json = json.dumps(supplier_barcode_json)
```
Comment (ivantodorovich) [inline pos_supplierinfo_barcode/models/product.py:17]: It'd be nice to have a simple unittest for this computed field

**Review by ivantodorovich — APPROVED** [review]
LGTM

**Pull request merged**